setup.py

```diff
@@ -2,6 +2,16 @@
 # Copyright (c) 2012-2022 Snowflake Computing Inc. All rights reserved.
 #
 
+import os
+
 from setuptools import setup
 
-setup()
+SQLALCHEMY_SRC_DIR = os.path.join("src", "snowflake", "sqlalchemy")
+VERSION = (1, 1, 1, None)  # Default
+with open(os.path.join(SQLALCHEMY_SRC_DIR, "version.py"), encoding="utf-8") as f:
```

Comment:
I could see this being useful, but how is it related to this telemetry change?

Reply:
we send the version number also part of the telemetry change and version number is used on the backend to check supported versions

Reply:
The method that composes telemetry info is already checked in but just not used:
https://github.com/snowflakedb/snowflake-sqlalchemy/blob/main/src/snowflake/sqlalchemy/util.py#L99